monai/bundle/config_item.py

```diff
@@ -177,6 +177,11 @@ class ConfigComponent(ConfigItem, Instantiable):
           but requires the dependencies to be instantiated/evaluated beforehand.
         - ``"_disabled_"`` (optional): a flag to indicate whether to skip the instantiation.
         - ``"_desc_"`` (optional): free text descriptions of the component for code readability.
+        - ``"_mode_"`` (optional): the operating mode for invoking the ``component``:
+
+            - ``"default"``: returns ``component(**kwargs)``
```

Comment:
Could you please help describe more about the "component" here, it can be a class name or function? It may cause unexpected behavior for users.

Thanks.